internal/storage/clients.go

```diff
@@ -106,9 +106,22 @@ func newS3Client(ctx context.Context, a *models.AwsS3) (*s3.Client, error) {
 					func(so *retry.StandardOptions) {
 						so.MaxAttempts = a.RetryMaxAttempts
 						so.MaxBackoff = time.Duration(a.RetryMaxBackoff) * time.Millisecond
-						so.Backoff = retry.NewExponentialJitterBackoff(
-							time.Duration(a.RetryBackoff) * time.Millisecond,
-						)
+						so.Backoff = retry.BackoffDelayerFunc(func(attempt int, _ error) (time.Duration, error) {
```

Comment:
`NewExponentialJitterBackoff` is the standard implementation and it should work.